tests/libtest/test_ibm_hci_nodes.py

```
@@ -295,5 +296,63 @@ def test_node_operations_with_ceph_health_check(self):
+            rack_ip = rack_data.get("rackInfo", {}).get("rackIP")
+            if not rack_ip:
+                logger.warning(f"Skipping rack {rack_serial}: no rackIP available")
+                continue
+
+            nodes_dict = rack_data.get("nodes", {})
+            for node_role in nodes_dict:
+                node_name = f"{node_role}.{rack_serial}.{domain_suffix}"
+                logger.info(f"Checking power status (direct) for: {node_name}")
+
+                status = ibm_hci_obj.power_status_direct(node_name)
+                logger.info(f"  {node_name} -> {status}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)